pandas/core/generic.py

```diff
@@ -7419,6 +7419,8 @@ def clip(
         squeeze : bool, default False
             Reduce the dimensionality of the return type if possible,
             otherwise return a consistent type.
+            deprecated:: 1.1.0
```

Comment:
```suggestion
            .. deprecated:: 1.1.0
```

Should put a newline before as well

Reply:
Thanks, I did that in the newest commit

Reply:
Did you push the latest changes? Doesn’t appear to be updated here on GitHub

Reply:
If I view the changes over all commits, there is an empty line after the description and before deprecated: 1.1.0

Reply:
Looks like this change still needs to be made